databricks/koalas/plot.py

```diff
@@ -1543,6 +1647,12 @@ def __call__(
         sort_columns=False,
         **kwds
     ):
+        plot_backend = _get_plot_backend(kwds.pop("backend", None))
+        # when using another backend, let the backend take the charge
+        if plot_backend.__name__ != "databricks.koalas.plot":
+            plot_data, kwds = _get_args_map(plot_backend.__name__, self.data, kind, kwds)
+            return plot_backend.plot(plot_data, x=x, y=y, kind=kind, **kwds)
```

Comment:
This doesn't read the other arguments. e.g.,

```py
kdf = ks.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8]], columns=["A", "B", "C", "D"])
kdf.plot(logx=True, backend="pandas_bokeh")
```

![Screen Shot 2020-07-09 at 3 05 11 PM](https://user-images.githubusercontent.com/506656/87095460-9e95ff00-c1f5-11ea-9ee1-d7cff789989f.png)

whereas pandas:

```py
kdf.to_pandas().plot(logx=True, backend="pandas_bokeh")
```

![Screen Shot 2020-07-09 at 3 06 41 PM](https://user-images.githubusercontent.com/506656/87095569-cf763400-c1f5-11ea-992a-6584822364fb.png)

Reply:
I was trying to work this out but now I feel we should NOT try to map arguments from one backend to another (which I was trying to, using `args_map`). So this would imply that: we pass the minimal important info to the `.plot` function and other arguments are passed using kwargs? Does this sound suitable?

Reply:
@DumbMachine, so are you saying we should handle known/important arguments and pass other arguments as other keyword arguments, instead of whitelisting everything? I think that way is okay.

We can fix or think about different ways later if that way is proven that it doesn't work in many other cases. I think it will work in most of important cases.